src/main/java/appeng/client/gui/implementations/GuiInterfaceTerminal.java

```diff
@@ -29,8 +29,14 @@
 import java.util.Set;
 import java.util.WeakHashMap;
 
+import appeng.container.slot.AppEngSlot;
+import appeng.container.slot.SlotCraftingMatrix;
+import appeng.container.slot.SlotFakeCraftingMatrix;
 import com.google.common.collect.HashMultimap;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Wrong order for 'com.google.common.collect.HashMultimap' import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)